### PR TITLE
Latexify Fig 23.1 Book2

### PR DESCRIPTION
**Pull request by Madhav-Kanda**
### Figure Number

Book2- Fig23.1

### Figures

| Before PR | After PR |
| :-: | :-: |
| image|image|

### Issue 

#93 

## Steps

- [x] Latexify

## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

### CODE REVIEW

File: notebooks/book2/23/flow_2d_mlp.ipynb [hunk 1]
```diff
@@ -0,0 +1,550 @@
+{
```
Comment (karm-patel) [inline notebooks/book2/23/flow_2d_mlp.ipynb:1]: <p>Can you figure out what is <code>xlabel</code> and <code>ylabel</code> for this plots? I think ylabel should be same for all, so in that case you can have a common ylabel.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (karm-patel) [inline notebooks/book2/23/flow_2d_mlp.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #8.        ax.scatter(x[:, 0], x[:, 1], s=10, cmap=cm, c=color)
</pre><p><code>s = 10</code> is too much for latexified figure, can you decrease it so we can see proper scatters in latexified figure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by karm-patel** [comment]
@Madhav-Kanda, After resolving these comments you can create a PR in the main repo.

File: notebooks/book2/23/flow_2d_mlp.ipynb (hunk 1 above)
Comment (Madhav-Kanda) [reply notebooks/book2/23/flow_2d_mlp.ipynb:1]: The plots depict sampling from various distributions. So should I add x1 and x2 as the label?

Comment (Madhav-Kanda) [reply notebooks/book2/23/flow_2d_mlp.ipynb:1]: After keeping s=1 I obtained this plot: 
image
Should I go ahead with this?

Comment (karm-patel) [reply notebooks/book2/23/flow_2d_mlp.ipynb:1]: scatter size looks good but in the first sub-plot `xtick_labels` are too closer. You can fix is setting only [-2.5, 2.5] or [-2, 0, 2] for first subplot only.

Comment (karm-patel) [reply notebooks/book2/23/flow_2d_mlp.ipynb:1]: After resolving this, you can directly create a PR in main repo.

**Pull request merged**